geode-docs/configuring/running/cluster-management-service.html.md.erb

```diff
@@ -0,0 +1,141 @@
+the License.  You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+**Note:** This feature is experimental and is subject to change in future releases of <%=vars.product_name_long%>.
+
+The cluster management service enables you to change the configuration of
+the cluster (for example, create/destroy regions, indexes, or gateway receivers/senders), and have these
+changes replicated on all the applicable servers and persisted in the cluster configuration service.
+
+With the cluster management service, you can:
+
+- Create regions/indexes on the fly.
+
+- Persist the configuration and apply it to the cluster, so when a new node joins, it has the configuration, and when the server restarts, it has the configuration.
+
+- Obtain a consistent view of the current configuration
+
+- Apply a change to all elements of the cluster in the same way
+
+- Be able to change the configuration in one place
+
+- Obtain this configuration without being on the cluster
+
+- Invoke the configuration service from the server or from a client
+
+The cluster management service is documented in the Geode Wiki: [Cluster Management Service](https://cwiki.apache.org/confluence/display/GEODE/Cluster+Management+Service).
+The Wiki write-up offers separate choices for the cluster management service implementation on each Geode version that supports it.
+
+The cluster management service is available as a [REST API](#cms-reference) and as a [Java API](#cms-java-api).
+
+## <a id='cms-reference'>Cluster Management Service REST API</a>
+
+The cluster management service REST API adheres to standard REST semantics, so users can use POST, PATCH, DELETE, and GET to create,
+update, delete, and read, respectively.
+
+This version of <%=vars.product_name%> uses the [1.11.0 Management REST API](https://cwiki.apache.org/confluence/display/GEODE/1.11.0+Management+REST+API+-+v1).
+As mentioned, the feature is experimental and still undergoing revision.
+
+For a definitive description of the cluster management service for your version of <%=vars.product_name%>, you can query the API directly to describe itself.
+The API returns a JSON body that contains a message describing the result along with standard HTTP status codes:
+
+1. Invoke `gfsh` on your current version of <%=vars.product_name%> and issue the `start locator` to start a locator using the default settings.
+2. Quit `gfsh` to return to the command prompt.
+3. Use `curl` to send a message asking the locator to describe its Cluster Management REST API. The API returns its description in JSON format. Pipe the output through the `jq` JSON formatting utility for readable output.
```

Comment:
Instead of telling them to use curl and jq, the easiest way for them to learn the API is to go to: http://localhost:7070/management/swagger-ui.html with a web browser.

Reply:
If they use a web browser then they have no need to quit the first gfsh. They can just stay in gfsh until done browsing and then to the shutdown command

Reply:
Replaced curl with Swagger